woocommerce/woocommerce
# Add new flow shipping recommendation task

### Changes proposed in this Pull Request:

Closes #33367

This PR:

1. Changes `Plugins` component in `@woocommerce/components` to have optional skip button since the step 2 is mandatory. https://github.com/woocommerce/woocommerce/commit/8200daf8a83809893fa5cd6ed2e53e55c445a41b
2. Adds link from payment settings page https://github.com/woocommerce/woocommerce/commit/5ee6243f0b9942236047744a8cdf753580f3b294
3. Adds a "hidden" tasklist, `shipping-recommendation` task, and its related fill in https://github.com/woocommerce/woocommerce/commit/0b7b4c6ed5316033f78c367321de958a1637f6a7

### How to test the changes in this Pull Request:

1. Enable the `shipping-smart-defaults` feature via WCA Test helper.
2. Set up a store in US
3. Go to WooCommerce > Settings > Shipping
4. Wait for "Recommended shipping solutions" to appear and click on "Get started" or "Activate" on WooCommerce Shipping
5. Observe it gets redirected to step 2 of `admin.php?page=wc-admin&task=shipping-recommendation`task
6. Click on "Install & enable" and observe `woocommerce-services` and `jetpack` plugins are installed
7. Click on "Connect" on the next step and connect site to WPCOM
8. When redirected back, observe the task automatically redirects to `admin.php?page=wc-settings&tab=shipping&section=woocommerce-services-settings`

<img width="764" alt="image" src="https://user-images.githubusercontent.com/3747241/176069679-5c1ca9b3-c5c8-4d29-a89a-ed71983cfa1b.png">


### Other information:




### FOR PR REVIEWER ONLY:
